### DISCUSSION TITLE
Directus Schema for testing

### DISCUSSION THREAD

**Discussion by casualmatt** [Intevel/nuxt-directus#17] [Ideas]
Hello to all, 

To simplify test and merge request check/contribution  it could be a good idea to share the[ Directus Schema](https://docs.directus.io/reference/cli/#migrate-schema-to-a-different-environment). 

I just want to install the #16 with my Directus to simplify my setup but i couldn't, ad to use the https://swyi1ut9.directus.app/ provided with the code.

This could also add security issue, to expose credential to a Directus instance, that could be use fraudulently.

**Comment by Intevel** [comment]
Hey, you're basically right. It makes sense to build an instance for testing. Besides, I will disable access to the data in the playground due to security issues.
